### ISSUE TITLE
Succeed/Test036.ML is currently failing

### ISSUE THREAD

**Issue by Earnestly** [polyml/polyml#87]
* linux 4.13.9
* gcc 7.2.0
* libffi 3.2.1

Currently https://github.com/polyml/polyml/commit/31b5de8ee56a091281390a9ce32d224f255911fc is resulting in the following test failure:

```
Test072.ML => Passed
Test073.ML => Done
Passed
Test074.ML => Passed

Failed Tests: Succeed/Test036.ML
make[2]: *** [Makefile;1175: check-local] Error 1
make[2]: Leaving directory '/home/earnest/build/pkgbuilds/polyml-git/src/polyml'
make[1]: *** [Makefile;997: check-am] Error 2
make[1]: Leaving directory '/home/earnest/build/pkgbuilds/polyml-git/src/polyml'
make: *** [Makefile;706: check-recursive] Error 1
```

Due to https://github.com/polyml/polyml/commit/c2a296122426f5a resulting in a build failure, I cannot bisect the repo.  Although prior commits are good.

```
Use: basis/IntInf.sml
Error- in './basis/IntInf.sml', line 115.
Type error in function application. Function: LargeInt.quot : int * int -> int Argument: (j, 2) : Int.int * FixedInt.int Reason: Can't unify int (*In Basis*) with Int.int (*In Basis*) (Different type constructors)
make[2]: *** [Makefile;1143: polyexport.o] Error 1
make[2]: Leaving directory '/home/earnest/build/pkgbuilds/polyml-git/src/polyml'
make[1]: *** [Makefile;706: all-recursive] Error 1
make[1]: Leaving directory '/home/earnest/build/pkgbuilds/polyml-git/src/polyml'
make: *** [Makefile;470: all] Error 2
```

**Comment by dcjm** [comment]
Thank you for pointing that out. I've fixed it in 99ce50d.  It all seems to work now.

**Comment by Earnestly** [comment]
Thanks
